### PR TITLE
added SMAP option to create_fv3_mapping and changed namelist

### PR DESCRIPTION
**Pull request by zcstanley**
I don't know if you actually want to merge this PR because I made destructive changes to the namelist, since the namelist is now written for SMAP/EASE grid and not for IMS.

### CODE REVIEW

File: create_fv3_mapping.f90 [hunk 1]
```diff
@@ -20,6 +20,10 @@ program create_fv3_mapping
   character*100      :: ims_lat_name = "imslat_4km_8bytes.bin"
   character*100      :: ims_lon_name = "imslon_4km_8bytes.bin"
 
+! SMAP input info
+  character*100      :: smap_path = "/scratch2/BMC/gsienkf/Zofia.Stanley/projects/fv3_mapping/"
```
Comment (ClaraDraper-NOAA) [inline create_fv3_mapping.f90]: I'll move this file, and email you a new location.

File: create_fv3_mapping.f90 [hunk 2]
```diff
@@ -20,6 +20,10 @@ program create_fv3_mapping
...
+  character*100      :: smap_file_name = "NSIDC0772_LatLon_EASE2_M09km_v1.0.nc"
```
Comment (ClaraDraper-NOAA) [inline create_fv3_mapping.f90]: Please change this to smal_latlon_name

File: fv3_mapping.nml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 &fv3_mapping_nml
```
Comment (ClaraDraper-NOAA) [inline fv3_mapping.nml]: Please revert this back to the original.

**Review by ClaraDraper-NOAA — CHANGES_REQUESTED** [review]

File: create_fv3_mapping.f90 (hunk 2 above)
Comment (zcstanley) [reply create_fv3_mapping.f90]: Done.

File: fv3_mapping.nml (hunk 3 above)
Comment (zcstanley) [reply fv3_mapping.nml]: Done.

**Comment by zcstanley** [comment]
I found out that no output was being written to the netcdf file for SMAP. I switched the order of i and j in the definition of tile, tile_i, and tile_j, which at least makes it so that the program writes data to netcdf. @ClaraDraper-NOAA, we should talk at our next meeting to determine if this quick fix is how we want to solve the issue. 

Note: I got an error about incorrect dimension sizes, which is how I found this. It likely did not come up with IMS because `source_i_size` = `source_j_size`, while for SMAP `source_i_size` =/= `source_j_size`

**Comment by zcstanley** [comment]
The good news: I have validated the output from fv3_mapping with SMAP's EASE grid. There are no missing values, the tiles look good, and the difference between EASE lat/lon and FV3 lat/lon is small. (See attached figures.)

The bad news: I had to switch i and j indices in the definition of the variables for the netcdf files. This will likely impact the interpretation of output from the current fv3_mapping. Note that the current fv3_mapping runs without errors for IMS because the IMS grid has the same number of grid points in each dimension. This is not true for the EASE grid, so the i,j switch was necessary to get the code to run without errors. 

EASE_latlon_vs_FV3_tile
EASE_lat_vs_FV3_lat
EASE_lon_vs_FV3_lon

**Comment by ClaraDraper-NOAA** [comment]
Thanks Zofia. Let's discuss tomorrow about the switching of i, j. You shouldn't have to make any changes to the IMS (or generic) parts of the code.

File: create_fv3_mapping.f90 (hunk 1 above)
Comment (zcstanley) [reply create_fv3_mapping.f90]: Done.

**Comment by ClaraDraper-NOAA** [comment]
Please check that the index file works OK for IMS with the dimensions switched. This looks like a bug in the original code (not caught because IMS dimensions are the same for i and j)

**Comment by zcstanley** [comment]
I confirmed that the IMS test still passes with the switched (corrected) i & j indices.

**Pull request merged**